daisy/workflow.go

```diff
@@ -259,6 +259,7 @@ func (w *Workflow) cleanup() {
 			w.LogWorkflowInfo("Error returned from cleanup hook: %s", err)
 		}
 	}
+	w.LogWorkflowInfo("Workflow %q finished clean up.", w.Name)
```

Comment:
minor: to be consistent with other logs, either use "cleaning up" (preferred) or "cleanup".